.travis.yml

```diff
@@ -8,3 +8,7 @@ before_script:
   - cd src/nih_wayfinding
   - npm install
   - bower install
+env:
+  global:
+  - secure: aUCLz8UqbeWRs4neCsXWRkd8lw5x5BsERGuPCxUaeNnocpzXLCy4KHTFqPMfeV34bFJQ9K5CIGCURM2oVT8++WrMkHs9qTY8OJ64jP+qs9IpUhGsMqXWzG58cdwz44WggarwNSWKgkGHLk+1IuqvZNdEqkFTmvaLi1Tbi/NahpU=
```

Comment:
There's nothing exposed; those are encrypted key/value pairs, made using the `travis` command-line tool, [as recommended](https://docs.saucelabs.com/ci-integrations/travis-ci/).